### ISSUE TITLE
Introduce Lint Checks 

### ISSUE THREAD

**Issue by razajafri** [NVIDIA/spark-rapids-jni#1656]
We should introduce lint checks that prevent files with format problems like line-length checks as was caught by @ttnghia in #1623

_Originally posted by @ttnghia in https://github.com/NVIDIA/spark-rapids-jni/pull/1623#discussion_r1428235581_

**Comment by ttnghia** [comment]
An alternative way is to enforce line length in our IntelliJ style: https://www.jetbrains.com/help/idea/code-style-sql-code.html#:~:text=To%20configure%20the%20line%20length,wrap%20at%20N%20columns%20field.

**Comment by razajafri** [comment]
> An alternative way is to enforce line length in our IntelliJ style: https://www.jetbrains.com/help/idea/code-style-sql-code.html#:~:text=To%20configure%20the%20line%20length,wrap%20at%20N%20columns%20field.

Yes, but that won't work for people using vscode. We should do both like we do in the plugin
